test/release/examples/primers/associative.good

```diff
@@ -1,9 +1,9 @@
-associative.chpl:72: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
-associative.chpl:306: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
-associative.chpl:307: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
-associative.chpl:309: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
-associative.chpl:314: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
-associative.chpl:315: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
+associative.chpl:68: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
```

Comment:
Do we want to strip line numbers from these errors to avoid future updates like this, or are we testing that these line numbers are accurate? If we are, the primers don't seem like the right place for that kind of testing....